### PR TITLE
Weather App

### PR DESCRIPTION
**Pull request by Smart1-hub**
Choose an API to base the development of the webapp

    Home page
        When the page loads, the webapp shows the list of categories that could be filtered out by some parameter
        Along with the category name, some numeric values per category that come from the API  are displayed.
        When the user clicks (or taps) on a category item, the application navigates to the details page.

    Details page
        In the details page, the webapp retrieves data from the API to show detailed data in the category.
        When the user clicks on the "Back" button (<), the user navigates to the home page.
Create unit tests for pure functions in your Redux code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,69 @@
-# Getting Started with Create React App
+## Weather-App
+A weather application that displays weather updates of different capital cities.
+The weather updates is fetched from [Open Weather API](https://openweathermap.org/api).
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## Preview
+![screenshot](./images/screenshot.png)
```
Comment (Cathella) [inline README.md]: ℹ️ This project image fails to load in the README. Kindly reference it well.

File: README.md [hunk 2]
```diff
@@ -1,70 +1,69 @@
...
 
-## Available Scripts
+## Project Presentation
 
-In the project directory, you can run:
+- [Presentation](https://drive.google.com/file/d/1x8L418Dmo94-Zcl_GiO9QlTXD9oiZ3x7/view?usp=sharing)
```
Comment (Cathella) [inline README.md]: ✅ The student recorded a video presenting the features of the project 👍🏻

File: README.md [hunk 3]
```diff
@@ -1,70 +1,69 @@
...
+- React
+- Redux
+- Github
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Live Link
 
-### `npm test`
+[Click here](https://deploy-preview-3--monumental-paletas-7ab791.netlify.app)
```
Comment (Cathella) [inline README.md]: ✅ The student deploys the application using Heroku, Netlify or GitHub Pages 👍🏻

File: README.md [hunk 4]
```diff
@@ -1,70 +1,69 @@
...
+- Twitter: [@ademola_adebayo](https://twitter.com/ademola_adebayo)
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+- LinkedIn: [Ademola Adebayo](https://www.linkedin.com/in/ademola-adebayo-81051578/)
 
-## Learn More
+## Acknowledgement
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+- Original design idea is by Nelson Sakwa in Behance.
```
Comment (Cathella) [inline README.md]: ✅ The README contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/) 👍🏻

File: src/Redux/configureStore.js [hunk 5]
```diff
@@ -9,4 +9,4 @@ const rootReducer = combineReducers({
 });
 const store = createStore(rootReducer, applyMiddleware(thunk));
 
-export default store;
\ No newline at end of file
+export default store;
```
Comment (Cathella) [inline src/Redux/configureStore.js]: ✅ It connects to Redux store, using the `connect()` function or [Redux hooks](https://react-redux.js.org/api/hooks) 👍🏻

**Review by Cathella — CHANGES_REQUESTED** [review]
## 🟡 Status: Changes Required 🛑

### Hi @Smart1-hub 👋🏻, 

I love the precise design skills exhibited in this project 😍, how you explicitly explain how you went through with the overall development with the components 💯 and the attention to detail. The major issue is the length of the project video. It has a very long introduction which makes it longer.

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## To Highlight 👏🏻

✅ No linter errors.
✅ You used correct Git and GitHub Flow.

## Required Changes ♻️

- [ ] _Check the comments under the review._
- [ ] I kindly request that you re-record the video and try to make the introduction shorter so that it is accommodated within the 5 required minutes.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Smart1-hub — COMMENTED** [review]
Update reviewer's comments on read-me and project presentation video

**Review by zhadier — APPROVED** [review]
## Hi @Smart1-hub  👋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 

### Keep Rocking!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**